components/label.scss

```diff
@@ -0,0 +1,6 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/label";
+@import "../scss/icons/icon";
```

Comment:
Do we need icons passed along with labels?

Reply:
Not with this component. 👍